lib/DiscoveryStreamFeed.jsm

```diff
@@ -331,15 +340,24 @@ this.DiscoveryStreamFeed = class DiscoveryStreamFeed {
     }
 
     if (!layout || !layout.layout) {
-      if (this.config.hardcoded_basic_layout) {
+      if (
+        this.config.hardcoded_basic_layout ||
+        Services.prefs.getBoolPref(PREF_HARDCODED_BASIC_LAYOUT, false)
```

Comment:
Why check both instead of only the standalone pref?

Reply:
I figured if either of them were set, we would be pretty safe to enable the basic layout, right?